pegen/grammar_parser.py

```diff
@@ -28,7 +28,7 @@ class GeneratedParser(Parser):
 
     @memoize
     def start(self):
-        # start: rules $ { Grammar ( rules ) }
+        # start: rules $ {'Grammar ( rules )'}
```

Comment:
I don't like this change -- there must be something in the generator that causes this change.